src/coreclr/vm/argdestination.h

```diff
@@ -83,6 +83,37 @@ class ArgDestination
 #endif // !DACCESS_COMPILE
 #endif // defined(TARGET_ARM64)
 
+#if defined(TARGET_LOONGARCH64)
+    bool IsStructPassedInRegs()
+    {
+        return m_argLocDescForStructInRegs != NULL;
+    }
+
+#ifndef DACCESS_COMPILE
+    void CopyStructToRegisters(void *src, int fieldBytes)
+    {
+        _ASSERTE(IsStructPassedInRegs());
+        _ASSERTE(m_argLocDescForStructInRegs->m_cFloatReg == 1);
```

Comment:
Where is the case of two floating registers handled?

Reply:
> Where is the case of struct passed in two floating registers handled?

A struct containing just one floating-point real is passed as though it were a standalone floating-point real.

This case is the same with the RISC-V: https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-cc.adoc